pkg/overlay/cache.go

```diff
@@ -19,6 +19,8 @@ import (
 const (
 	// OverlayBucket is the string representing the bucket used for a bolt-backed overlay dht cache
 	OverlayBucket = "overlay"
+	// IsOnlineWindow is the maximum amount of time that can pass without seeing a node before that node is considered offline
+	IsOnlineWindow = 1 * time.Hour
```

Comment:
Great! I just renamed it to `OnlineWindow`. The "is" prefix in names is usually used for boolean flags and even often omitted for them. It also reads better this way, e.g. "if the duration between now and last contact success time is less than the online window...".

Reply:
sounds good!